src/qrules/io/_dot.py

```diff
@@ -497,14 +497,15 @@ def _collapse_graphs(
         )
         for g in graphs
     }
+
```

Comment:
Try to keep the diff small. Ideally, any changes (even whitespace stuff) should be either removed or motivated.